Listener/MaintenanceListener.php

```diff
@@ -194,7 +194,10 @@ public function onKernelRequest(GetResponseEvent $event)
         }
 
         $route = $request->get('_route');
-        if (null !== $this->route && preg_match('{'.$this->route.'}', $route)  || (true === $this->debug && '_' === $route[0])) {
+        if ($route && (
```

Comment:
I don't think that this is correct, is it? See #95.

Reply:
it is covered by test and this approach is used in our project.